chef_master/source/backend_failure_recovery.rst

```diff
@@ -153,3 +153,35 @@ Once you've resolved the issue and can run the ``pgsql-follow`` command successf
 
    rm /var/opt/chef-backend/leaderl/data/unsynced
    chef-backend-ctl start leaderl
+
+General Follower Recovery Process
+=====================================
+
+Initial attempts to recover should follow this general pattern and use the scenarios and tools shown above to assist in the recovery steps:
+
+#. With the cluster down, take a filesystem level backup of all backend nodes.
+#. Check the health of the leader and repair if necessary. If the `/var/log/chef-backend/leaderl/current` logs do not show leadership changes and the ``chef-backend-ctl cluster-status`` shows a solid etcd/pgsql leader, then you are good to continue. 
+
+   **Note**: Any leadership repair process is a very dangerous step that can result in data loss. We recommend `opening a ticket with Support <www.chef.io/support/get-started>`__ to go over any cluster leadership issue.
+   
+   Any leadership repair process often involves removing an internal pgsql lock file that prevents promotion of what is thought as the last leader. This file is placed automatically by leaderl when it demotes a leader ``/var/opt/chef-backend/leaderl/data/no-start-pgsql``. Refer to the `Promoting a Previous Leader section </backend_failure_recovery.html#promoting-a-previous-leader>`__ for more details. 
+#. If necessary, promote what is thought as the most recent leader. Refer to the `Promoting a Previous Leader section </backend_failure_recovery.html#promoting-a-previous-leader>`__ for more detail.
+#. Sync the followers from the leader using a full basebackup because the WAL entries have likely already rotated. When this happens, the followers will complain in ``/var/log/chef-backend/postgresql/X.Y/current`` about being unable to sync. Using just the `--recovery` flag will result in timeouts of the ``chef-backend-ctl join-cluster`` command. It's impossible for a follower to sync and rejoin while in this state because it doesn't have current enough info.
```

Comment:
What is "this" in the second sentence "When this happens"? Syncing? Or entry rotation?

Reply:
That should be "When the WAL entries have already rotated away the followers will complain"